lib/appbundler/cli.rb

```diff
@@ -9,10 +9,12 @@ class CLI
     banner(<<-BANNER)
 * appbundler #{VERSION} *
 
-Usage: appbundler APPLICATION_DIR BINSTUB_DIR
+Usage: appbundler BUNDLE_DIR BINSTUB_DIR [GEM_NAME] [GEM_NAME] ...
```

Comment:
did you just break https://github.com/chef/appbundle-updater and https://github.com/chef/kitchen-appbundle-updater here?   they'll probably both need updates and its possible that the kitchen-tests will go red in travis if you push this before fixing them.